addons/account/models/account_move.py

```diff
@@ -264,7 +260,7 @@ def _get_default_invoice_incoterm(self):
         default=lambda self: self.env.user)
     invoice_date = fields.Date(string='Invoice/Bill Date', readonly=True, index=True, copy=False,
         states={'draft': [('readonly', False)]},
-        default=_get_default_invoice_date)
+        default=False)
```

Comment:
remove the `default` keyword instead

Reply:
What about keeping the default method, but have it return None instead of Today?

Reply:
We don't want to keep useless methods indefinitely. Deprecating in stable and removing in master is better IMO.

Reply:
I tested this on an existing database - pulled the code and restarted Odoo, no module upgrades.
The result is the same behavior as before.
Modifying this default method to return None would make this change effective without the need to upgrade `account` (not a small thing to do).
The method would still be slated for deprecation in 15.0, of course.

Reply:
@dreispt That is not true, the default value is only a runtime thing.
Tried it myself too just to be  100000% sure even if I was already 99.99% sure.

Reply:
@william-andre Hmm, OK, I still see the default date in my test database, I don't know why. I'll have to hunt that down to get to the bottom of it.

Reply:
@william-andre You're right. It was a set issue on my side. Test looks good.